### PR TITLE
Add Swiper carousels to code.org/videos and code.org/minecraft pages

### PR DESCRIPTION
**Pull request by kelbyhawn**
Updates the https://code.org/videos and https://code.org/minecraft pages to use Swiper carousels.
- This applies the work done in this PR https://github.com/code-dot-org/code-dot-org/pull/58054
- The Minecraft page shares the `carousel_inspirational_video` view so I needed to update that page too, apologies for the PR bloat!
- To test the fallback player append this to the url: `?force_youtube_fallback=1`

**Followup:** I'm going to add available S3 backup videos to the carousels in a separate ticket [ACQ-1802](https://codedotorg.atlassian.net/browse/ACQ-1802)

**Jira ticket:** [ACQ-1359](https://codedotorg.atlassian.net/browse/ACQ-1359)

----

## Videos page

https://github.com/code-dot-org/code-dot-org/assets/9256643/a00dd730-528f-4d44-98c0-db501b9e7203

## Minecraft page

https://github.com/code-dot-org/code-dot-org/assets/9256643/65b84644-4085-43fe-bc53-16587c970a7e

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/educate/resources/videos.haml [hunk 1]
```diff
@@ -42,7 +42,7 @@ video_player: true
       - if index === 5
         = view :carousel_inspirational_video
       - if index === 6
-        = view :carousel_cs_video
+        = view :carousel_careers_in_cs_video
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/educate/resources/videos.haml:45]: I renamed this to be more descriptive.

File: pegasus/sites.v3/code.org/views/carousel_how_ai_works_video.haml [hunk 2]
```diff
@@ -1,22 +1,19 @@
...
 
-.carousel-wrapper#how-ai-works-videos
-  .carousel.carousel--flexible_units
-    .slides
-      - video_ai_index.times do |index|
-        .slide
-          %figure.flex-space-between.video-wrapper.video-wrapper--one-col
-            .video__container
-              = view :display_video_thumbnail, id: video_id_second[index], video_code: video_ai_first[index], play_button: 'center', letterbox: 'false', caption: video_ai_caption[index]
+- video_download_path = ["//videos.code.org/ai_series/HowAIWorks_LLM_SM_CoBrand.mp4", "//videos.code.org/ai_series/HowAIWorks_Creativity_SM_CoBrand.mp4", "//videos.code.org/ai_series/ai-01-intro-to-ai.mp4", "//videos.code.org/ai_series/ai-02-machine-learning.mp4", "//videos.code.org/ai_series/ai-03-training-data.mp4", "//videos.code.org/ai_series/ai-04-neural-networks.mp4", "//videos.code.org/ai_series/ai-05-computer-vision.mp4", "//videos.code.org/ai_series/ai-06-ai-ethics-access-bias.mp4", "//videos.code.org/ai_series/ai-07-ai-ethics-privacy-work.mp4"]
```
Comment (bethanyaconnor) [inline pegasus/sites.v3/code.org/views/carousel_how_ai_works_video.haml:3]: Just so I understand -- adding these links to more carousels is what the task you created is for? I noticed when playing around with it locally that most of the carousels still use YouTube even with the URL param `force_youtube_fallback=1` but it sounds like that's expected?

**Review by bethanyaconnor — APPROVED** [review]
This looks so good!! Thank you for all your work here. I have one clarifying question but otherwise LGTM!

File: pegasus/sites.v3/code.org/views/carousel_how_ai_works_video.haml (hunk 2 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/views/carousel_how_ai_works_video.haml:3]: Correct! My best guess for what happens if YouTube is blocked and there's no download backup is that it'll show the default "YouTube is blocked" (paraphrasing) message.

**Pull request merged**